src/cargo/core/workspace.rs

```diff
@@ -722,9 +724,10 @@ impl<'gctx> Workspace<'gctx> {
                         continue;
                     }
                     bail!(
-                        "package `{}` is listed in workspace’s default-members \
-                         but is not a member.",
-                        path.display()
+                        "package `{}` is listed in default-members\n\
+                        for workspace at {} but is not a member.",
```

Comment:
```suggestion
                        "package `{}` is listed in default-members but is not a member\n\
                        for workspace at {}.",
```
This keeps the need-to-parse part of the message up front and people can more easily glance at the rest of the message